docs/src/@primer/gatsby-theme-doctocat/components/layout.js

```diff
@@ -0,0 +1,154 @@
+import {
```

Comment:
It feels really unfortunate to  duplicate this whole component. Maybe @colebemis could upstream the idea of supporting multiple links?

Reply:
One other, in the middle idea. What if the section where we currently have the "View source example" was refactored to be a `LinksList` component. Then we could override just that more targeted component to render our links.

Reply:
I'd ❤️  to have this directly in doctocat! But I don't think it is a big issue to have it here while it isn't supported

Reply:
👋 I opened a PR to upstream this into Doctocat: https://github.com/primer/doctocat/pull/220

I also included some design updates.